src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,65 @@
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+<%
+Conversation conversation = (Conversation) request.getAttribute("conversation");
+List<Message> messages = (List<Message>) request.getAttribute("messages");
+%>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Activity Feed</title>
+  <link rel="stylesheet" href="/css/main.css" type="text/css">
+
+  <style>
+    #chat {
+      background-color: white;
+      height: 500px;
+      overflow-y: scroll
+    }
+  </style>
+
+  <%-- <script>
+    // scroll the chat div to the bottom
+    function scrollChat() {
+      var chatDiv = document.getElementById('chat');
+      chatDiv.scrollTop = chatDiv.scrollHeight;
+    };
+  </script> --%>
+</head>
+<body onload="scrollChat()">
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else { %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
```

Comment:
u dont need the nav bar. u can use <jsp:include page="/WEB-INF/includes/header.jsp"/> instead so changes to the nav bar are global